app/app.go

```diff
@@ -81,8 +81,6 @@ import (
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
```

Comment:
In this file, we can also remove `testdata.RegisterQueryServer(app.GRPCQueryRouter(), testdata.QueryImpl{})`